File: src/schema/LazyRequest.ts
export interface LazyRequest {
	guild_id: string;
	channels?: Record<string, [number, number]>;
	activities?: boolean;
	threads?: boolean;
	typing?: true;
	members?: any[];
}

export const LazyRequest = {
	guild_id: String,
	$activities: Boolean,
	$channels: Object,
	$typing: Boolean,
	$threads: Boolean,
	$members: [] as any[],
};

File: extra/admin-api/Models/Spacebar.Models.Gateway/LazyRequest.cs
using System.Text.Json.Serialization;

namespace Spacebar.Models.Gateway;

public class LazyRequest
{
    [JsonPropertyName("guild_id")]
    [JsonNumberHandling(JsonNumberHandling.AllowReadingFromString | JsonNumberHandling.WriteAsString)]
    public long GuildId { get; set; }

    // key is string because json...
    [JsonPropertyName("channels")]
    public Dictionary<string, List<List<int>>> Channels { get; set; }

    [JsonPropertyName("members")]
    public bool Members { get; set; }

    [JsonPropertyName("threads")]
    public bool Threads { get; set; }

    [JsonPropertyName("activities")]
    public bool Activities { get; set; }

    [JsonPropertyName("typing")]
    public bool Typing { get; set; }
}